### PR TITLE
React hydration errors fix

### PR DESCRIPTION
**Pull request by hernandoagf**

### CODE REVIEW

File: modules/esm/components/ProgressRing.tsx [hunk 1]
```diff
@@ -50,13 +48,7 @@ const ProgressRing = ({
...
         ></circle>
         <text x="50%" y="48%" textAnchor="middle" fill="#434358" fontSize="18px" dy=".3em">
-          {totalStaked ? (
-            `${formatValue(totalStaked, 'wad', 6)}      `
-          ) : (
-            <Box pl="14px" pr="14px">
-              <div ref={loader} />
-            </Box>
-          )}
+          {totalStaked ? `${formatValue(totalStaked, 'wad', 6)}      ` : '---      '}
```
Comment (b-pmcg) [inline modules/esm/components/ProgressRing.tsx:51]: is this supposed to have that many spaces after the ---?

File: modules/home/components/ResourcesLanding/ResourcesLanding.tsx [hunk 2]
```diff
@@ -6,29 +6,34 @@ SPDX-License-Identifier: AGPL-3.0-or-later
...
 import { ExternalLink } from 'modules/app/components/ExternalLink';
 import { fadeIn } from 'lib/keyframes';
 import { alpha } from '@theme-ui/color';
 
 const CategoryButton = ({ label, color, active, onClick }) => {
+  const { theme } = useThemeUI();
+  const [badgeSelectorColor, setBadgeSelectorColor] = useState('#000');
+
+  useEffect(() => {
+    setBadgeSelectorColor(theme.rawColors?.badgeSelector as string);
```
Comment (b-pmcg) [inline modules/home/components/ResourcesLanding/ResourcesLanding.tsx:21]: if `theme.rawColors?.badgeSelector` is undefined maybe we should pass the default value in

File: modules/esm/components/ProgressRing.tsx (hunk 1 above)
Comment (hernandoagf) [reply modules/esm/components/ProgressRing.tsx:51]: Seems like the format is still correct without the extra spaces, I'll remove those

**Review by b-pmcg — APPROVED** [review]
🚀

**Comment by tyler17** [comment]
Do you have an example exec where the statbox looked weird?

**Comment by hernandoagf** [comment]
> Do you have an example exec where the statbox looked weird?

@tyler17 The UI is ok, but it would throw a React hydration error on the console because `div` tags cannot appear as children of `p` tags

**Comment by tyler17** [comment]
Ah got it, thanks! All looks good to me then.

**Review by tyler17 — APPROVED** [review]

**Pull request merged**